### PR TITLE
Show hint text everywhere it exists

### PR DESCRIPTION
**Pull request by issyl0**
Screenshot 2020-03-23 at 08 43 22
Screenshot 2020-03-23 at 08 43 39

### CODE REVIEW

**Review by theseanything — APPROVED** [review]

**Comment by issyl0** [comment]
I don't like the spacing between the title and the hint text on the name question. 🤔

File: app/views/coronavirus_form/name.html.erb [hunk 1]
```diff
@@ -15,6 +15,10 @@
   margin_top: 0
 } %>
 
+<%= render "govuk_publishing_components/components/hint", {
```
Comment (leenagupte) [inline app/views/coronavirus_form/name.html.erb:25]: Should this be inside the form_tag?

Comment (issyl0) [reply app/views/coronavirus_form/name.html.erb:25]: Ah, I didn't copy 100% correctly from `app/views/contact_details.html.erb`. Yes, both the title and the hint are inside the form tag there. 😴One moment...

**Comment by alex-ju** [comment]
> I don't like the spacing between the title and the hint text on the name question. 🤔

You can set a `margin_bottom: 3` on L16 if you want the spacing to be similar with the one on pages with radios.

**Comment by timpaul** [comment]
> I don't like the spacing between the title and the hint text on the name question. 🤔

Agreed! Let's shrink the headings first and check again - we can apply spacing override classes if we need to

File: app/views/coronavirus_form/name.html.erb (hunk 1 above)
Comment (issyl0) [reply app/views/coronavirus_form/name.html.erb:25]: Done in b932ce2.

**Review by leenagupte — APPROVED** [review]

**Pull request merged**